File: 2st/Yeonwoo/원소들의 곱과 합.py
def solution(num_list):
    answer = 0
    sum_ = sum(num_list)**2
    multiply_ = 1
    for i in num_list:
        multiply_ *= i
    if multiply_ < sum_:
        answer = 1
    elif multiply_ > sum_:
        answer = 0
    return answer

File: 4st/Jisoo/programmer/원소들의 곱과 합.java
class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
        int p=1;
        for(int i : num_list){
            answer+=i;
            p*=i;
        }
        if(p<answer*answer){
            return 1;
        }
        else{
            return 0;
        }
    }
}